### ISSUE TITLE
List widget a11y attributes

### ISSUE THREAD

**Issue by chrmarti** [Microsoft/vscode#57274]
The list widget uses 'tree' and 'treeitem' roles which makes NVDA read it out as such and makes it mention the tree level which is always 1.

The focused item does not have `"aria-selected"="true"` which makes NVDA read out "not selected". The tree sets that attribute, maybe the list should just do that too. (Should test with the multi-select QuickPick.)

**Comment by cleidigh** [comment]
Related:
#53923
